<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pb_enter">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>120</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Ввести</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_map">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>400</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Проложить маршрут</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sb_n">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>42</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sb_m">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>42</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>80</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_wall">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>102</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Стена</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_portal">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>270</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Гребанный портал</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_hwall">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>240</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Высокая стена</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="grap">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>540</width>
      <height>540</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_trap">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>300</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Шипы, ёк макарёк</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sb_trap">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>330</y>
      <width>42</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
